dGame/dComponents/VendorComponent.cpp

```diff
@@ -129,12 +101,40 @@ void VendorComponent::SetupConstants() {
 	auto* vendorComponentTable = CDClientManager::Instance().GetTable<CDVendorComponentTable>();
 	std::vector<CDVendorComponent> vendorComps = vendorComponentTable->Query([=](CDVendorComponent entry) { return (entry.id == componentID); });
 	if (vendorComps.empty()) return;
-	m_BuyScalar = vendorComps[0].buyScalar;
-	m_SellScalar = vendorComps[0].sellScalar;
-	m_RefreshTimeSeconds = vendorComps[0].refreshTimeSeconds;
-	m_LootMatrixID = vendorComps[0].LootMatrixIndex;
+	auto vendorData = vendorComps.at(0);
+	m_BuyScalar = vendorData.buyScalar;
```

Comment:
lets correct this one to use the dZoneManager::Instance()->GetWorldConfig()->sell_thingy_idk_the_name

Reply:
I'm not 100% sure if that's correct, it looks like they may have been attempting to use the ones on the vendor component themselves rather than in worldconfig. Maybe worldconfig ones would be a default if none was provided

Reply:
The live client physically does not use the sell or buy values in the db.  It only uses the worldconfig.

Reply:
ah cool

Reply:
I stand correected.  I swear I looked up buyscalar and found no results.  Your presumptions about it defaulting to worldconfig was correct.